src/layouts/Header/components/SeatDetails/SeatDetails.tsx

```diff
@@ -0,0 +1,22 @@
+import { useSelfHostedSeatsConfig } from 'services/selfHosted'
+
+function SeatDetails() {
+  const { data: selfHostedSeats } = useSelfHostedSeatsConfig({
+    suspense: false,
+  })
+
+  if (!selfHostedSeats?.seatsUsed && !selfHostedSeats?.seatsLimit) {
```

Comment:
should this be an || instead of &&?

Reply:
Maybe we can default another text here instead of null 🤔 

Just spitballing

Reply:
Honestly just copied the old one lol

You may be right though will have a think about this and update. Any ideas?